manifests/database.pp

```diff
@@ -0,0 +1,27 @@
+# Set up the pulp database
+class pulp::database {
+  exec { 'migrate_pulp_db':
+    command     => 'pulp-manage-db',
+    path        => '/bin:/usr/bin',
+    logoutput   => 'on_failure',
+    user        => 'apache',
+    refreshonly => true,
+    require     => File['/etc/pulp/server.conf'],
+  }
+
+  if $pulp::reset_data {
```

Comment:
You could remove this -- its legacy that we never removed when porting our original module.